dplython/dplython.py

```diff
@@ -72,8 +72,16 @@ def ungroup(self):
     self._grouped_on = None
     self._grouped_self = None
 
+  def regroup(self, names):
+    self.group_self(names)
+    return self
+
   def apply_on_groups(self, delayedFcn):
-    if isinstance(delayedFcn, mutate) or isinstance(delayedFcn, sift):
+    # use names and a list instead of isinstance here to keep things manageable?
+    # if everything goes to verbs, can we even remove the apply on groups altogether?
```

Comment:
The hope is to indeed eventually remove some of this code. It might be unavoidable for some functions though. However, can you remove this comment-- probably shouldn't be in the source. Feel free to ask questions like this in the open issues or on the not-yet-very-utilized-but-hopefully-soon-it-will-be [mailing list](https://groups.google.com/forum/#!forum/dplython).